src/test/java/nextstep/subway/line/domain/SectionsTest.java

```diff
@@ -0,0 +1,188 @@
+package nextstep.subway.line.domain;
+
+import nextstep.subway.line.exception.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class SectionsTest {
+
+    private Long 역1;
+    private Long 역2;
+    private Long 역3;
+    private Long 역4;
+
+    @BeforeEach
+    void setUp() {
+        역1 = 1L;
+        역2 = 2L;
+        역3 = 3L;
+        역4 = 4L;
+    }
+
+    @Nested
+    class WhenShow {
+        @DisplayName("모든 구간에 해당하는 역 Id을 반환한다")
+        @Test
+        void getLastStationIdTest() {
+            //Given 역 4개가 있을때
+            var sections = new Sections();
+            sections.add(new Section(역1, 역2, 10L));
+            sections.add(new Section(역2, 역3, 10L));
+            sections.add(new Section(역3, 역4, 10L));
+
+            //when 모든 구간을 조회하면
+            var sectionIds = sections.getAllStationIds();
+
+            // Then 역id 를 반환한다
+            assertThat(sectionIds).containsOnly(1L, 2L, 3L, 4L);
+        }
```

Comment:
이렇게 표현하는게 더 올바를 것 같아요!
```suggestion
            // Then 역id 를 반환한다
            assertThat(sectionIds).containsOnly(역1, 역2, 역3, 역4);
        }
```